climpred/bootstrap.py

```diff
@@ -393,6 +394,99 @@ def varweighted_mean_period_threshold(control, sig=95, iterations=500, time_dim=
     )
 
 
+def _bootstrap_hindcast_over_init_dim(
+    hind,
+    hist,
+    verif,
+    dim,
+    resample_dim,
+    iterations,
+    metric,
+    comparison,
+    alignment,
+    compute,
+    reference_compute,
+    resample_uninit,
+    reference_alignment,
+    **metric_kwargs,
+):
+    """Helper function for looping bootstrap skill.
```

Comment:
Please replace this docstring with this, if you don't mind. Just for us to remember why we are splitting this out like this...

```
"""Bootstrap hindcast skill over the ``init`` dimension.

    When bootstrapping over the ``member`` dimension, an additional dimension
    ``iteration`` can be added and skill can be computing over that entire
    dimension in parallel, since all members are being aligned the same way.
    However, to our knowledge, when bootstrapping over the ``init`` dimension,
    one must evaluate each iteration independently. I.e., in a looped fashion,
    since alignment of initializations and target dates is unique to each
    iteration.

    See ``bootstrap_compute`` for explanation of inputs.
    """
```